css/templates/themes/2/page-vars.css

```diff
@@ -50,7 +50,7 @@
   --color-navbar-primary: var(--color-black);
   --color-navbar-font-primary: var(--color-white);
 
-  --color-input-primary: var(--color-font-accent);
+  --color-input-primary: var(--color-secondary);
```

Comment:
So I should keep " --color-input-primary: var(--color-font-accent);"  and add a new line: " --color-input-secondary: var(--color-secondary); " , right?